### DISCUSSION TITLE
Link Automatic Action to Metadata Field

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#648] [Feature Requests]
Is there a possibility to link the automatic action (for example: 'set to draft') from a CPT with and with a standard offset to a custom meta date field?

Example:

I'm using Jet Engine to create a CPT and attached a meta date field to it. This date field for example is the launch date of an event. So naturally, I want this CPT to be set to draft (in my case) when this meta date expires with (in my case) an offset of +1 day.

I know this can be done manually, but just like you can have a automatic action with an offset of the publishdate, to be able to attach it to a custom meta date field instead of the publishdate would open a world of possibilities.

If you would have a PHP snippet for this that would be awesome too.

Hope to hear from you guys!

Original request: https://secure.helpscout.net/conversation/2470370237/218149?viewId=6932143

**Comment by stevejburge** [comment]
Solve with "Metadata Scheduling"
